contracts/interfaces/IRibbon.sol

```diff
@@ -9,9 +9,34 @@ interface IRibbonVault {
 
     function cap() external view returns (uint256);
 
+    function WETH() external view returns (address);
+
+    function withdrawals(address) external view returns (uint256);
```

Comment:
Btw this function signature is wrong, it should be 

```solidity
struct Withdrawal {
        uint16 round;
        uint128 shares;
}

function withdrawals(address) external view returns (Withdrawal);
```

Reply:
Good catch!